src/i18n/I18nProvider.js

```diff
@@ -0,0 +1,35 @@
+import Polyglot from 'node-polyglot';
+import { Component, Children, PropTypes } from 'react';
+
+class I18nProvider extends Component {
```

Comment:
I'd call it `TranslationProvider` instead of `'I18nProvider`, because I never know which letters to capitalize in "i18n"